persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -24,8 +24,18 @@ deleteUser:
 DELETE FROM User WHERE qualified_id = ?;
 
 insertUser:
-INSERT OR REPLACE INTO User(qualified_id, name, handle, email, phone, accent_id, team, connection_status, preview_asset_id, complete_asset_id)
-VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
+INSERT INTO User(qualified_id, name, handle, email, phone, accent_id, team, connection_status, preview_asset_id, complete_asset_id)
+VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+ON CONFLICT(qualified_id) DO UPDATE SET
+name = excluded.name,
+handle = excluded.handle,
+email = excluded.email,
+phone = excluded.phone,
+accent_id = excluded.accent_id,
+team = excluded.team,
+connection_status = excluded.connection_status,
+preview_asset_id = excluded.preview_asset_id,
+complete_asset_id = excluded.complete_asset_id;
```

Comment:
`INSERT OR REPLACE` have the same result as updating all the values, if there is a value that should not be updated when an already stored user is inserted you need to remove the `excluded` for that value.

Reply:
Great, my bad 😅